File: Tugas2/java/Ram.java
public class Ram{
    private String capacityr;
    private int pricer;

    // construct
    public Ram() {}

    public Ram(String capacityr, int pricer){
        this.capacityr = capacityr;
        this.pricer = pricer;
    }
	
	//set and get
	public void setcapacityr(String capacityr){
		this.capacityr = capacityr;
	}
	public String getcapacityr(){
		return this.capacityr;
	}
	
	public void setpricer(int pricer){
		this.pricer = pricer;
	}
	public int getpricer(){
		return this.pricer;
	}

    public void printRam(){
        System.out.println("Kapasitas : " + getcapacityr());
		System.out.println("Harga : " + getpricer());
    }

}

File: Tugas2/php/ram.php
<?php

class ram
{
	//Attribute
	private $capacityr = "";
	private $pricer = 0;
	
	
	//Construct
	public function __construct($capacityr = "", $pricer = 0){
		$this->capacityr = $capacityr;
		$this->pricer = $pricer;
	}
	
	//Set dan get
	public function setcapacityr($capacityr){
		$this->capacityr = $capacityr;
	}
	public function getcapacityr(){
		return $this->capacityr;
	}
	
	public function setpricer($pricer){
		$this->pricer = $pricer;
	}
	public function getpricer(){
		return $this->pricer;
	}

    //Output
    public function printram(){
		echo "Kapasitas : ". $this->getcapacityr(). "<br/>";
		echo "Harga : ". $this->getpricer(). "<br/>";
	}
}
?>